opm/simulators/wells/MultisegmentWell.hpp

```diff
@@ -122,6 +122,14 @@ namespace Opm
                                     WellState& well_state,
                                     Opm::DeferredLogger& deferred_logger) override;
 
+        virtual void assembleWellEqWithoutIteration(const Simulator& ebosSimulator,
```

Comment:
Please do not move this function to be `public`, at least not in this PR. Basically, `MultisegmentWell.hpp` should not be touched in this PR.